### DISCUSSION TITLE
Serial communication with DIY-Multiprotocol-TX-Module

### DISCUSSION THREAD

**Discussion by zygmunt** [pascallanger/DIY-Multiprotocol-TX-Module#493] [Q&A]
Good morning,

Here: [docs/Transmitters.md](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/docs/Transmitters.md) you can find info like:
_Transmitters that run er9X, erSky9X or OpenTx firmwares (like the FrSky Taranis, FlySky TH9X and Turnigy 9X family of transmitters) have the option of using a fast serial communication protocol between the Tx and the DIY Multiprotocol module ..._

Where I can find serial protocol description? Which will cover questions like:
How to select protocol which have to be used from Tx?
How data have to be organized and all other stuff which will help me control my model.

I'm asking because I want to use DIY-Multiprotocol-TX-Module in my Tx.

**Comment by kilrah** [comment]
https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/2098cdc5e04e22e84ceb8559ceb74b2dd5d83b48/Multiprotocol/Multiprotocol.h#L755
